app/src/main/java/org/pesmypetcare/mypetcare/services/StubCommunityService.java

```diff
@@ -35,13 +36,13 @@
     public static void addStubDefaultData() {
         addGroups();
         addTags();
-        addFroums();
+        addForums();
     }
 
     /**
      * Add the forums.
      */
-    private static void addFroums() {
+    private static void addForums() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method length is 17 lines (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5443034)